<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotControlDashboard</class>
 <widget class="QMainWindow" name="RobotControlDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="robotList">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>250</width>
      <height>581</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>581</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="launchButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addRobotButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>266</y>
      <width>170</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="cameraPortLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cameraPortValue">
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="streamingType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>83</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="hasStartStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Has Start/Stop</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>103</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="noCommands">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="hasCommands">
        <property name="text">
         <string>Yes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="streamingLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>396</y>
      <width>96</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Streaming Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="hasCameraGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>161</width>
      <height>58</height>
     </rect>
    </property>
    <property name="title">
     <string>Has Camera</string>
    </property>
    <widget class="QRadioButton" name="noCamera">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>35</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="hasCamera">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>35</y>
       <width>49</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>73</y>
      <width>222</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="robotNameLabel">
         <property name="text">
          <string>Robot Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="robotNameValue">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="ipLabel">
         <property name="text">
          <string>IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ipValue">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portValue"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="deleteRobotButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
